src/Demos/HSP.lagda

```diff
@@ -204,49 +176,31 @@ module _ {A : Type α }{B : A → Type β} where
 
 %% -----------------------------------------------------------------------------
 \subsection{Setoids}\label{setoids}
-A \defn{setoid} is a pair (\ab A, \af{≈}) where \ab A is a type and \af{≈}
-is an equivalence relation on \ab A. Setoids seem to have gotten a bad wrap
-in some parts of the interactive theorem proving community because of the extra
-overhead they require. However, we feel they are ideally suited to
-representing the basic objects of informal mathematics (i.e., sets)
-in a constructive, type-theoretic way.
-
-In informal mathematical discourse, a set typically comes equipped with an equivalence
-relation manifesting the notion of equality of elements of the set. We
-often take this equivalence for granted or view it as self-evident; rarely do we
-take pains to define it explicitly. While well-suited to informal
-mathematics, this approach is inadequate for formal, machine-checked proofs.
-
-The \agdaalgebras library was first developed without setoids, relying exclusively
-on the inductive equality type \ad{\au{}≡\au{}}, defined in \am{Agda.Builtin.Equality},
+A \defn{setoid} is a pair of a type \ab A and
```

Comment:
Nice revisions overall, but I would prefer, "...is a pair consisting of a type..."  (i.e., please add the word "consisting")

Reply:
Sure